BlitzBoard/Features/HomeScreenViewController.swift

```diff
@@ -1,16 +1,59 @@
+// LeagueTableViewController.swift
+// BlitzBoard
 //
-//  HomeScreenViewController.swift
-//  BlitzBoard
-//
-//  Created by Kiasha Rangasamy on 2024/05/06.
+// Created by Kiasha Rangasamy on 2024/04/11.
 //
 
 import UIKit
 
-class HomeScreenViewController: UIViewController {
-
+class LeagueTableViewController: UIViewController, ViewModelDelegate {
+    
+    func reloadView() {
+    }
+    
+    func show(error: String) {
+    }
+    
+    // MARK: IBOutlets
+    @IBOutlet private weak var tableView: UITableView!
+    @IBOutlet private weak var wallpapers: UIImageView!
+    private lazy var viewModel = LeagueTableViewModel(repository: LeagueTableRepository(), delegate: self)
+    
+    var leagueTable: [LeagueTableModel] = []
```

Comment:
is this being used?
If it is, it should be in the viewModel

Reply:
yes its being used
I checked, it actually is in the ViewModel already

Reply:
can I just remove it from here ?